src/scripts/hc_webhooks.js

```diff
@@ -0,0 +1,32 @@
+import { User, connect } from 'human-connection-api-nodejs-client';
+import mongoose from 'mongoose';
+import slugify from 'slugify';
+import constants from '../config/constants';
+
+export async function contributeProcedure(data) {
+  connect(constants.HC_BACKEND_URL);
+  const ProcedureModel = mongoose.model('Procedure');
+  const { procedureId, email, password } = data || {};
+  if (procedureId && email && password) {
+    const procedure = await ProcedureModel.findOne({ procedureId });
+    if (procedure) {
+      const user = new User({ email, password });
+      await user.contribute({
+        title: procedure.title,
+        content: `<p>${procedure.abstract}</p>`,
+        contentExcerpt: procedure.abstract,
+        type: 'post',
+        language: 'de',
+        categoryIds: ['5ac7768f8d655d2ee6d48fe4'], // politics & democracy
+      }, {
+        slug: slugify(procedure.title, { lower: true }),
```

Comment:
Slug should not be needed, if you provide none, our system will generate a unique one based on the title

Reply:
I know, but then I would have to store some sort of unique identifier (`slug` or `id`) which I would like to avoid. I don't want to modify the database of democracy